polars/tests/it/time/date_range.rs

```diff
@@ -1,34 +1,29 @@
-use polars::time::date_range;
```

Comment:
the changes in this file are mostly just whitespace, if reviewing I suggest ignoring whitespace: https://github.com/pola-rs/polars/pull/9554/files?w=1